README.md

```diff
@@ -1 +1,176 @@
+<h1>E-Pocket</h1>
 
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Video-Demo](Video-Demo)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [E-Pocket] <a name="about-project"></a>
+
+**[E-Pocket]**
+ this is mobile web application designed for budget management. The app enables users to handle their finances efficiently by organizing transactions linked to specific categories. This functionality offers users visibility into their spending patterns, allowing them to track expenses and understand where their money is allocated.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+     <li><a href="https://developer.mozilla.org/en-US/docs/Web/css">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Models]**
+- **[Controllers]**
+- **[Views]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git git@github.com:Nessrine88/My_Wallet.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec/models/model_name_spec.rb
+```
+
+### Deployment
+
+[E-Pocket](https://my-budget-ror-app.onrender.com/users/sign_up)
```

Comment:
- [ ] Good job so far, but there are still some changes needed. First of all, I noticed that you did not redeploy the updated code. I recommend redeploying with the latest commit. Additionally, in the existing deployed version, after signing in, when I try to create a category, it gives an error. As it is an internal server error, I failed to verify why it is happening. I recommend checking your deployed version and ensuring that you fix those issues.

https://github.com/Nessrine88/My-Budget-RoR-app/assets/55840999/0028c12f-b8a9-4374-a30a-58c598174ee6

- [ ] Please update your deployed link. 👍 
```suggestion

[E-Pocket](https://my-budget-ror-app.onrender.com)
```